src/web/modules/transactions/update/upload.njk

```diff
@@ -0,0 +1,37 @@
+{% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "common/errorSummary.njk" import errorSummary %}
+{% extends "layout/layout.njk" %}
+
+{% block main %}
+  <span class="govuk-caption-m">GOV.UK Pay platform</span>
+  <h1 class="govuk-heading-m">Update transactions</h1>
+
+  {% for message in messages %}
+    <div class="govuk-error-summary success-summary" role="alert">
+      <h2 class="govuk-error-summary__title">{{message}}</h2>
+    </div>
+  {% endfor %}
+
+  {% for error in errors %}
+    <div class="govuk-error-summary" role="alert">
+      <h2 class="govuk-error-summary__title">{{error}}</h2>
+    </div>
+  {% endfor %}
+
+  <form method="POST" action="/transactions/update?_csrf={{ csrf }}" enctype="multipart/form-data">
```

Comment:
csrf is included in the query string, as multer interferes with csurf and removes the csrf from the form data